qst/src/main/java/io/deephaven/qst/table/AggregateAllTable.java

```diff
@@ -35,7 +35,7 @@ public static Builder builder() {
     public static Optional<Aggregation> singleAggregation(
             AggSpec spec, Collection<? extends ColumnName> groupByColumns,
             Collection<? extends ColumnName> tableColumns) {
-        Set<ColumnName> exclusions = AggregateAllExclusions.of(spec, groupByColumns);
+        Set<ColumnName> exclusions = AggregateAllExclusions.of(spec, groupByColumns, tableColumns);
```

Comment:
I don't think we should move ColumnFormatting into qst; I'm thinking this should be the responsibility of the calling layer (engine). What about expanding this function to take in an additional parameter `Collection<? extends ColumnName> specialExclusions`?

Reply:
Then we won't get it right if we try to call `singleAggregation` from the QST layer.